src/api_routes_http.rs

```diff
@@ -124,11 +124,13 @@ use lemmy_apub::api::{
   search::search,
   user_settings_backup::{export_settings, import_settings},
 };
+use lemmy_routes::images::image_proxy;
 use lemmy_utils::rate_limit::RateLimitCell;
 
 pub fn config(cfg: &mut web::ServiceConfig, rate_limit: &RateLimitCell) {
   cfg.service(
     web::scope("/api/v3")
+      .route("/image_proxy", web::get().to(image_proxy))
```

Comment:
Not sure whats a good place for this, `/site` doesnt seem to fit. I also wanted to add some rate limiting with the code below, but then all other api calls would be routed through here and return 404.
```

        .service(
          web::scope("")
              .wrap(rate_limit.message())
              .route("/image_proxy", web::get().to(image_proxy)),
        )
```

Reply:
Should probably be in `crates/routes/src/images.rs`  instead of here.

And maybe a top level `/image/proxy?` Up to you tho.

Reply:
Its already defined in that file. And having it under /api/v3 makes more sense considering https://github.com/LemmyNet/lemmy/issues/4001.